Tests/GraphQL/HyperwalletTransferMethodConfigurationTests.swift

```diff
@@ -183,6 +130,91 @@ class HyperwalletTransferMethodConfigurationTests: XCTestCase {
         XCTAssertEqual(branchIdMask?.scrubRegex, "\\s")
     }
 
+    func testRetrieveTransferMethodConfigurationFeeAndProcessingTime_success() {
+        // Given
+        let request = setUpTransferMethodConfigurationRequest("TransferMethodConfigurationFeeAndProcessingTimeResponse")
```

Comment:
Update testRetrieveTransferMethodConfigurationFeeAndProcessingTime_success to whatever function name will be

Reply:
Renamed to `testRetrieveTransferMethodTypesFeesAndProcessingTimes`